datasets/msra_ner/msra_ner.py

```diff
@@ -0,0 +1,148 @@
+# coding=utf-8
```

Comment:
Could you only include chanegs related to `isizulu_ner_corpus ` in this PR please ?